LayoutTests/interaction-region/inline-link-in-layer-expected.txt

```diff
@@ -0,0 +1,42 @@
+This is a link, outside the layer.
+This is a link, inside the layer.
+(GraphicsLayer
+  (anchor 0.00 0.00)
+  (bounds 800.00 600.00)
+  (children 1
+    (GraphicsLayer
+      (bounds 800.00 600.00)
+      (contentsOpaque 1)
+      (drawsContent 1)
+      (backgroundColor #FFFFFF)
+      (event region
+        (rect (0,0) width=800 height=600)
+
+      (interaction regions [
+        (region
+            (rect (-3,-3) width=35 height=24)
+)
```

Comment:
I guess there's an indentation bug with region dumping

Reply:
There is! I keep meaning to fix it, but that belongs in another patch because it's going to change a lot of results.